subprojects/docs/src/docs/userguide/running-builds/gradle_daemon.adoc

```diff
@@ -258,9 +258,33 @@ This message means that either
 --
 In both cases the build cannot benefit from file system watching.
 
-`Caught exception: Couldn't add watch, error = 28`::
-If you receive this error on Linux then you ran out of inotify handles.
-To raise the limit see https://confluence.jetbrains.com/display/IDEADEV/Inotify+Watches+Limit[here].
-
 `java.io.IOException: Too many open files`::
 If you receive this error on macOS, you need to raise your open files limit, see https://superuser.com/a/443168/8117[here].
+
+===  Linux-specific notes
+
+File system watching uses http://en.wikipedia.org/wiki/Inotify[inotify] on Linux.
+Depending on the size of your build, it may be necessary to increase inotify limits.
+If you are using an IDE, then you probably already had to increase the limits in the past.
+
+[[sec:inotify_watches_limit]]
+Inotify watches limit::
```

Comment:
Now this remains the only element of this description list (or whatever `*:` is called). I think we can remove it.